swagger_server/itm/itm_ta1_controller.py

```diff
@@ -5,16 +5,29 @@
 from swagger_server.models.probe_response import ProbeResponse  # noqa: F401,E501
 from swagger_server.models.alignment_results import AlignmentResults  # noqa: F401,E501
 from swagger_server.models.alignment_target import AlignmentTarget  # noqa: F401,E501
+from swagger_server import config_util
+
 
 class ITMTa1Controller:
+    
+    config_util.check_ini()
+    config = config_util.read_ini()[0]
+    
     ADEPT_PORT = os.getenv("ADEPT_PORT")
     if (ADEPT_PORT is None or ADEPT_PORT == ""):
-        ADEPT_PORT = '8081'
+        ADEPT_PORT = config['DEFAULT']['ADEPT_PORT']
     SOARTECH_PORT = os.getenv("SOARTECH_PORT")
     if (SOARTECH_PORT is None or SOARTECH_PORT == ""):
-        SOARTECH_PORT = '8084'
-
-    def __init__(self, alignment_target_id, scene_type, alignment_target = None):
+        SOARTECH_PORT = config['DEFAULT']['SOARTECH_PORT']
+        
+    ADEPT_HOSTNAME = os.getenv("ADEPT_HOSTNAME")
+    if (ADEPT_HOSTNAME is None or ADEPT_HOSTNAME == ""):
+        ADEPT_HOSTNAME = config['DEFAULT']['ADEPT_HOSTNAME']
+    SOARTECH_HOSTNAME = os.getenv("SOARTECH_HOSTNAME")
```

Comment:
I wasn't sure if replacing the environment variables was a good idea because of the way docker boots up the environment. I figured falling back to the config file if no environment is set was a safer bet to make sure systems that are in place don't have to be touched.

I can remove the environment variables entirely if you think the redundancy is unecessary.